### PR TITLE
Wip new

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
needs review

@AnnJohn @jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,111 @@
...
+
+New York City, for example, has a robust API called NYC Open Data. Developers can connect to this API to find information about city programs, public housing, parks, schools, construction, you name it. 
+
+When you send a request for data to such an API, the data you get back from them will be in the form of a nested hash that can contain information about thousands of records. 
+
+We'll learn a lot more about APIs later on in this course. For now, just understand that nested hashes are a very common occurrence in programming, they are used to store complex collections of data and you will encounter them in particular when working with APIs, among other places.
+
+### Nested Hashes Model Real-World Data
+
+We can imagine so many real-world situations and environments in which we are dealing with complicated collections of data. Take the example of a simple address book.
```
Comment (jmburges) [inline README.md]: Tie the example into some sort of NYC OPen Data. Maybe like NYC Open Data has a directory of doctors or w/e and link to it. Trying to really tie it to the real wordl

File: README.md [hunk 2]
```diff
@@ -1 +1,111 @@
...
+]
+```
+
+This works, but it's not a great model of a real address book. In a real address book, contacts are sorted, or grouped, usually by last name. In our collection of hashes above, there is no way to organize our data like that. 
+
+This is where nested hashes come in. 
+
+## Our First Nested Hash
+
+With a nested hash, we can create a key that points to a value of *another hash or an array*. Let's take a look:
```
Comment (jmburges) [inline README.md]: Mention that remember anything can be a value in a hash. So! hashes are included in anything :)

File: README.md [hunk 3]
```diff
@@ -1 +1,111 @@
...
+			{last_name: "DeBenedetto", first_name: "Zoe", phone: "917-555-9875"}, 
+			{last_name: "DeBenedetto", first_name: "Richard", phone: "917-555-0912"}
+		  ], 
+	"L" => [
+			{last_name: "Londin", first_name: "Victoria", phone: "917-555-0912"}
+		  ]
+}
+```
+
+Nested hashes allow us to further group, or associate, the data we are working with. They help us to deal with situations in which a category or piece of data is associated not just to one discrete value, but to a collection of values. In such a situation, we can create a hash key that points to a value of another hash or an array.
```
Comment (jmburges) [inline README.md:65]: love this explanation

File: README.md (hunk 2 above)
Comment (jmburges) [inline README.md]: jk should have just kept reading

File: README.md [hunk 4]
```diff
@@ -1 +1,111 @@
...
+You may have noticed in the above examples that we used both an array of hashes and a hash in which the value of a key is an array. 
+
+Understand that arrays and hashes can store *any type of data*. In other words, the individual index items of an array can be strings, integers, even other arrays and hashes. The same is true of hashes. The values that hash keys point to may be strings, integers, even arrays and hashes. 
+
+
+## Nested Hash Example
+
+We'll be building up our own nested hash and operating on such data structures in a number of ways over the course of the next few lessons. For now, just read through the next example and get comfortable looking at a nested hash. 
+
+In this example, we have a hash, `flatiron_school`, which stores some data about us. This data is broken down into the categories of `:instructors`, `:dev_team` and `:students`, thanks to our nested hash.
```
Comment (jmburges) [inline README.md:79]: You now have three different "domains" you've talked about. Maybe bring this teacher domain up into the previous example?

**Comment by jmburges** [comment]
looks good, be careful with bringing in too many domains. Try and stick to just one within this reading. Maybe the teacher thing?

**Comment by SophieDeBenedetto** [comment]
@jmburges addressed all comments.

**Comment by jmburges** [comment]
Add in a note under the link to the JSON from NYC Open Data that if you view this using the [Chrome JSON Viewer extension](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc?hl=en) you'll see that it looks just like a nested hash! It's really hard to see that the json is a nested hash without that viewer.

**Comment by jmburges** [comment]
do that and :shipit: 

Also when you make changes and want me to take a look at them, don't forget to assign it to meeeeeeee!

**Pull request merged**